packages/dropdowns/stories/examples/Menu/Default.tsx

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story } from '@storybook/react';
+import { Dropdown, Menu, Trigger, Item } from '@zendeskgarden/react-dropdowns';
+import { Button, IconButton } from '@zendeskgarden/react-buttons';
+import SettingsIcon from '@zendeskgarden/svg-icons/src/16/gear-stroke.svg';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+
+interface IStoryProps {
+  placement:
+    | 'auto'
+    | 'top'
+    | 'top-start'
+    | 'top-end'
+    | 'bottom'
+    | 'bottom-start'
+    | 'bottom-end'
+    | 'end'
+    | 'end-top'
+    | 'end-bottom'
+    | 'start'
+    | 'start-top'
+    | 'start-bottom';
+  hasArrow: boolean;
+  isAnimated: boolean;
+  isCompact: boolean;
+  isDisabled: boolean;
+}
+
+export const Default: Story<IStoryProps> = ({
+  hasArrow,
+  isAnimated,
```

Comment:
`isAnimated` is not working correctly on initial render. It probably needs to be set to true in `Default.args` in order to match the API.